tmt/steps/provision/__init__.py

```diff
@@ -519,11 +759,18 @@ def _ssh_socket(self) -> str:
         return self._ssh_socket_path
 
     def _ssh_options(self, join: bool = False) -> Union[str, List[str]]:
-        """ Return common ssh options (list or joined) """
+        """ Return common ssh options (list or joined)
+            Rationale for default ServerAlive* settings:
+            There is issue with reboots so this seems to be issues with high values
+            for reboot. However, some tests needs to run really long (kernel recompilation
+            for example), so user need to be able to change those options.
+        """
         options = [
             '-oForwardX11=no',
             '-oStrictHostKeyChecking=no',
             '-oUserKnownHostsFile=/dev/null',
+            '-oServerAliveInterval=60',
+            '-oServerAliveCountMax=5',
             ]
```

Comment:
IIUIC, this is where we would `+ self.ssh_option`, to propagate options given on the command line (command-line => `data.ssh_option` => `guest.ssh_option`).